sdcm/sct_runner.py

```diff
@@ -503,24 +499,43 @@ def _image(self, image_type: ImageType = ImageType.SOURCE) -> Any:
 
         return aws_region.resource.Image(existing_amis[0]["ImageId"])
 
-    # pylint: disable=too-many-arguments
+    # pylint: disable=too-many-arguments,too-many-locals
     def _create_instance(self,
                          instance_type: InstanceTypeType,
                          base_image: Any,
                          tags: dict[str, str],
                          instance_name: str,
                          root_disk_size_gb: int = 0,
-                         region_az: str = "",
+                         region_name: str = "",
```

Comment:
why are you changing the signature of the function here ?

Reply:
it's causing change to all other implementation which isn't needed now

let's revert that part

Reply:
running provision-aws